core/domain/feedback_domain.py

```diff
@@ -17,9 +17,59 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+
+import datetime
+
 from core import utils
 from core.domain import user_services
 
+from typing import Dict, List, Optional
+from typing_extensions import TypedDict
+
+
+class FeedbackThreadDict(TypedDict):
+    """Object defining a domain object for a feedback thread"""
```

Comment:
```suggestion
    """Dictionary representing the FeedbackThread object."""
```

ditto below (in similar fashion)